web/components/cycles/sidebar.tsx

```diff
@@ -129,7 +129,7 @@ export const CycleDetailsSidebar: React.FC<Props> = observer((props) => {
           title: "Restore success",
           message: "Your cycle can be found in project cycles.",
         });
-        router.push(`/${workspaceSlug.toString()}/projects/${projectId.toString()}/cycles/${cycleId}`);
+        router.push(`/${workspaceSlug.toString()}/projects/${projectId.toString()}/archives/cycles`);
```

Comment:
here as well, please check all the router.push in this PR are required. remove them if not required